src/mono/netcore/System.Private.CoreLib/src/System/Globalization/GlobalizationMode.Mono.cs

```diff
@@ -6,12 +6,18 @@ namespace System.Globalization
 {
     internal static partial class GlobalizationMode
     {
-        private static bool GetInvariantSwitchValue()
+        private static bool GetInvariantSwitchValue() =>
+            GetSwitchValue("System.Globalization.Invariant", "DOTNET_SYSTEM_GLOBALIZATION_INVARIANT");
+
+        private static bool GetSwitchValue(string switchName, string envVariable)
```

Comment:
You don't need `switchName` for mono, right? since mono doesn't support runtime switches, just the env var name?

Reply:
Fixed